App/api/PushController .js

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect} from 'react';
+import {View, Button} from 'react-native';
+import PushNotification from 'react-native-push-notification';
+
+export const PushController = () => {
```

Comment:
Just use service for it and invoke setup method of this service in useEffect (root component or HomeScreen)